daprdocs/content/en/getting-started/quickstarts/resiliency/resiliency-state-quickstart.md

```diff
@@ -0,0 +1,900 @@
+
+{{< tabs "Python" "JavaScript" ".NET" "Java" "Go" >}}
+ <!-- Python -->
+{{% codetab %}}
+
+### Pre-requisites
+
+For this example, you will need:
+
+- [Dapr CLI and initialized environment](https://docs.dapr.io/getting-started).
+- [Python 3.7+ installed](https://www.python.org/downloads/).
+<!-- IGNORE_LINKS -->
+- [Docker Desktop](https://www.docker.com/products/docker-desktop)
+<!-- END_IGNORE -->
+
+### Step 1: Set up the environment
+
+Clone the [sample provided in the Quickstarts repo](https://github.com/dapr/quickstarts/tree/master/resiliency).
+
+```bash
+git clone https://github.com/dapr/quickstarts.git
+```
+
+In a terminal window, navigate to the `order-processor` directory.
+
+```bash
+cd ../state_management/python/sdk/order-processor
+```
+
+Install dependencies
+
+```bash
+pip3 install -r requirements.txt 
+```
+
+### Step 2: Run the application with resiliency enabled
+
+Run the `order-processor` service alongside a Dapr sidecar. In the `dapr run` command below the `--config` parameter applies a Dapr configuration that enables the resiliency feature.
+
+```bash
+dapr run --app-id order-processor ../config.yaml --components-path ../../../components/ -- python3
+```
+
+The resilency spec is:
+- Located in the `components` directory. 
+- Automatically discovered by the Dapr sidecar when run in standalone mode.
+
+   ```yaml
+   apiVersion: dapr.io/v1alpha1
+   kind: Resiliency
+   metadata:
+     name: myresiliency
+   scopes:
+     - checkout
+   
+   spec:
+     policies:
+       retries:
+         retryForever:
+           policy: constant
+           maxInterval: 5s
+           maxRetries: -1 
+   
+       circuitBreakers:
+         simpleCB:
+           maxRequests: 1
+           timeout: 5s 
+           trip: consecutiveFailures >= 5
+   
+     targets:
+       apps:
+         order-processor:
+           retry: retryForever
+           circuitBreaker: simpleCB
+   ```
+
+Once the application has started, the `order-processor`service writes and reads `orderId` key/value pairs to the `statestore` Redis instance [defined in the `statestore.yaml` component]({{< ref "statemanagement-quickstart.md#statestoreyaml-component-file" >}}).
+
+```bash
+== APP == Saving Order:  { orderId: '1' }
+== APP == Getting Order:  { orderId: '1' }
+== APP == Saving Order:  { orderId: '2' }
+== APP == Getting Order:  { orderId: '2' }
+== APP == Saving Order:  { orderId: '3' }
+== APP == Getting Order:  { orderId: '3' }
+== APP == Saving Order:  { orderId: '4' }
+== APP == Getting Order:  { orderId: '4' }
+```
+
+### Step 3: Introduce a fault
+
+Simulate a fault by stopping the Redis `statestore.yaml` instance that was initalized when executing `dapr init` on your development machine. Once the instance is stopped, write and read operations from the order-processor service begin to fail.
+
+Since the `resiliency.yaml` spec defines the `statestore.yaml` component as a target, all failed requests will apply retry and circuit breaker policies:
```

Comment:
```suggestion
Since the `resiliency.yaml` spec defines the `statestore` as component target, all failed requests will apply retry and circuit breaker policies:
```